addon/components/frost-select-dropdown.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
     ]
       .join(';')
 
-    return Ember.String.htmlSafe(style)
+    return EmberString.htmlSafe(style)
```

Comment:
I am not crazy about the `EmberString`, `EmberObject` and `EmberRouter` approach. It feels forced just so that we can destructure but then the need to use an alias makes it less clean. I think that in these cases it is more clear to just use `Ember.String`, `Ember.Object` and `Ember.Router`.